src/current/cockroachcloud/production-checklist.md

```diff
@@ -66,6 +66,20 @@ When several transactions try to modify the same underlying data concurrently, t
 
 To ensure optimal SQL performance for your CockroachDB {{ site.data.products.cloud }} cluster, follow the best practices described in the [SQL Performance Best Practices](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/performance-best-practices-overview) guide.
 
+## Configure node shutdown settings
+
+Clusters in CockroachDB {{ site.data.products.cloud }} have the following default values for [cluster settings]({% link {{ site.current_cloud_version }}/cluster-settings.md %}) that relate to [node shutdown]({% link {{ site.current_cloud_version }}/node-shutdown.md %}) for maintenance, upgrades, or scaling. Depending on the requirements of your applications and workloads, you may need to modify `server.shutdown.connections.timeout` or `server.shutdown.transactions.timeout`.
+
+Cluster setting | Default | Details
+----------------|---------|---------
+[`server.shutdown.initial_wait`]({% link {{ site.current_cloud_version }}/cluster-settings.md %}#setting-server-shutdown-drain-wait)<br />Alias: `server.shutdown.drain_wait` | 3 minutes | How long to wait before redirecting new client requests to non-draining nodes. After this duration has elapsed, the node no longer accepts new client connections. This setting cannot be changed for clusters in CockroachDB {{ site.data.products.cloud }}.
+[`server.shutdown.connections.timeout`]({% link {{ site.current_cloud_version }}/cluster-settings.md %}#setting-server-shutdown-connection-wait)<br />Alias: `server.shutdown.connection_wait` | 0 seconds | How long after `server.shutdown.initial_wait` completes to wait for client connections to drain before forcibly disconnecting them from the node. Can be set to a value between 0 and 30 minutes (1800 seconds).
+[`server.shutdown.transactions.timeout`]({% link {{ site.current_cloud_version }}/cluster-settings.md %}#setting-server-shutdown-query-wait)<br />Alias: `server.shutdown.query_wait` | 90 seconds | How long to wait for transactions on the node to complete before rolling them back. The default value is appropriate for most deployments. If this is set too low, transactions will be rolled back unnecessarily during node shutdown, and if it is set too high, node shutdown may take longer.
```

Comment:
We should note that 90s is the highest supported value. Increasing it beyond this value can result in tx failures during cluster operations.